problems/arrayMethods.js

```diff
@@ -64,15 +98,23 @@ const numbersOnly = () => {};
  * @returns {boolean} Whether or not all elements are the same.
  */
 
-const isAllSame = () => {};
+const isAllSame = (items) => {
+  return items.every((item) => {
+    return items[0] === item
+  })
+};
 
 /**
  * Takes in an array and returns whether or not all elements in the array are odd.
  * @param {number[]} nums
  * @returns {boolean} All odd?
  */
 
-const isAllOdd = () => {};
+const isAllOdd = (nums) => {
+  return nums.every((num) => {
+    return num % 2 === 1
```

Comment:
This is perfect!

Another way to write it is "return num % 2" without "=== 1" because 1, like all numbers besides 0, is truthy!

Just another way of thinking!